docs/esignet-local.postman_environment.json

```diff
@@ -0,0 +1,135 @@
+{
+	"id": "6713547a-2a22-47eb-bc0c-ebb39ea837dd",
```

Comment:
For the file naming, if possible, let follow hyphen separated, that way it will be consistent with other files in the docs section

Reply:
Done